ODLA/platforms/odla_popart/test/odla_ops_math_test.cc

```diff
@@ -808,12 +808,115 @@ TEST_CASE("MATH OPS TESTING") {
 
     }
 
-    SUBCASE("MATH OPS AND TEST") 
-    {}
-    SUBCASE("MATH OPS OR TEST")
-    {}
+    SUBCASE("MATH OPS AND TEST") {
+    odla_computation comp;
+    CHECK_EQ(ODLA_SUCCESS, odla_CreateComputation(&comp));
+    set_computationItem(comp, 1);
+
+    auto lhs = odla_CreateArgument({ODLA_BOOL, {.size = 2, .dims = {2, 2}}},
+                                    (const odla_value_id)("lhs"));
+
+    auto rhs = odla_CreateArgument({ODLA_BOOL, {.size = 2, .dims = {2, 2}}},
+                                    (const odla_value_id)("rhs"));
+
+    auto And = odla_And(lhs, rhs, (const odla_value_id) "And");
+    odla_SetValueAsOutput(And);
+
+    static odla_context ctx;
+    odla_CreateContext(&ctx);
+
```

Comment:
I suggest you that replace data with  lhs_data[F, F, T, T] and rhs_data[T, F, T, F] to make "or" "and" "not equal" op more test case